Makefile.docker

```diff
@@ -9,6 +9,7 @@ docker-cilium-image-for-developers:
 	$(QUIET)DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE) build \
 		$(DOCKER_FLAGS) \
 		--build-arg LOCKDEBUG=\
+		--build-arg RACE=${RACE}\
```

Comment:
When `RACE=1` , here we get `LOCKDEBUG=` and `RACE=1` which won't be honored that `${RACE}` imply `${LOCKDEBUG}` right?

I would be fine with `--build-arg LOCKDEBUG=${RACE}` so that it is explicit that both are either enabled or disabled by the same flag.

Reply:
Good catch!